articles/hardware/hs64/ts4231.md

```diff
@@ -3,10 +3,28 @@ uid: hs64_ts4231
 title: Headstage64 Position Data
 ---
 
-The following excerpt from the Headstage64 [example workflow](xref:hs64_hs64) demonstrates electrical stimulation.
+The following excerpt from the Headstage64 [example workflow](xref:hs64_hs64) demonstrates the Headstage64's position
+tracking capabilities.
 
-::: workflow
+::: workflow 
 ![/workflows/hardware/hs64/ts4231.bonsai workflow](../../../workflows/hardware/hs64/ts4231.bonsai)
 :::
 
-The <xref:OpenEphys.Onix1.TS4231V1PositionData> operator generates a sequence of <xref:OpenEphys.Onix1.TS4231V1PositionDataFrame>s.
\ No newline at end of file
+The <xref:OpenEphys.Onix1.TS4231V1PositionData> operator generates a sequence of
+[TS4231V1PositionDataFrames](xref:OpenEphys.Onix1.TS4231V1PositionDataFrame). These `TS4231V1PositionDataFrames` are
+emitted when a receiver on the Headstage64 successfully receives a fully formed set of signals from both lighthouses.
+With no occlusions, the maximum achievable sample rate is 30 Hz. There are three receivers on the Headstage64, so it's
```

Comment:
With no occlusions, the maximum achievable sample rate is 30 Hz for each receiver.